google/resource_endpoints_service.go

```diff
@@ -111,6 +124,20 @@ func resourceEndpointsService() *schema.Resource {
 	}
 }
 
+func migrateEndpointsService(v int, is *terraform.InstanceState, meta interface{}) (*terraform.InstanceState, error) {
```

Comment:
Usually we have these in a separate file, which might also be a bit more future-proof for when this file becomes generated.

Reply:
Done.